src/libs/firebase/remoteConfig/helpers/transformDefaultValues.ts

```diff
@@ -0,0 +1,9 @@
+export const transformDefaultValues = (values: object): Record<string, string> => {
```

Comment:
il me semble que généralement, on préfère typer plus spécifiquement que `object`

d'après les exemples que je vois dans les tests, ça serait

```suggestion
export const transformDefaultValues = (values: Record<string, number | boolean | string | Record<string, unknown> | Array<number | boolean | string>): Record<string, string> => {
```

ou si on ne sait pas

```suggestion
export const transformDefaultValues = (values: Record<string, unknown>): Record<string, string> => {
```